non-ipynb/effect-size/commands.sh

```diff
@@ -1,24 +1,24 @@
-# curl -o ag_map_with_alpha.txt ftp://ftp.microbio.me/AmericanGut/manuscript-package/1250/ag_map_with_alpha.txt
-# curl -o data_dictionary.csv ftp://ftp.microbio.me/AmericanGut/manuscript-package/data_dictionary.csv .
+curl -o ag_map_with_alpha.txt ftp://ftp.microbio.me/AmericanGut/manuscript-package/1250/ag_map_with_alpha.txt
+curl -o data_dictionary.csv ftp://ftp.microbio.me/AmericanGut/manuscript-package/data_dictionary.csv .
 
 # open data_dictionary and replace
-# :%s/^M/\r/g
-# :%s/alzhemiers/alzheimers/g
+:%s/^M/\r/g
+:%s/alzhemiers/alzheimers/g
 
-# ./dataExtractor.sh ag_map_with_alpha.txt alpha_columns.txt > alpha_div.tsv
+./dataExtractor.sh ag_map_with_alpha.txt alpha_columns.txt > alpha_div.tsv
 
-# ./clean_map_agp.py --mapping ag_map_with_alpha.txt --data-dictionary data_dictionary.csv --output maps1000
-# for f in `find ./ -iname '*quartiles*' | grep './/maps'`; do echo $f; head -n 1 $f | awk '{print NF}'; done
-#   .//maps1000/ag_map_with_alpha.txt.quartiles.tsv
-#   361
-#   .//maps2000/ag_map_with_alpha.txt.quartiles.tsv
-#   149
-#   .//maps5000/ag_map_with_alpha.txt.quartiles.tsv
-#   126
+./clean_map_agp.py --mapping ag_map_with_alpha.txt --data-dictionary data_dictionary.csv --output maps1000
+for f in `find ./ -iname '*quartiles*' | grep './/maps'`; do echo $f; head -n 1 $f | awk '{print NF}'; done
+  # .//maps1000/ag_map_with_alpha.txt.quartiles.tsv
+  # 361
+  # .//maps2000/ag_map_with_alpha.txt.quartiles.tsv
+  # 149
+  # .//maps5000/ag_map_with_alpha.txt.quartiles.tsv
+  # 126
 
-./effect-size_v2.py --mappings maps1000/ag_map_with_alpha.txt.quartiles.tsv --alphas alpha_div.tsv --output alpha1000/
-./effect-size_v2.py --mappings maps2000/ag_map_with_alpha.txt.quartiles.tsv --alphas alpha_div.tsv --output alpha2000/
-./effect-size_v2.py --mappings maps5000/ag_map_with_alpha.txt.quartiles.tsv --alphas alpha_div.tsv --output alpha5000/
+./effect-size.py --mappings maps1000/ag_map_with_alpha.txt.quartiles.tsv --alphas alpha_div.tsv --output alpha1000/
+./effect-size.py --mappings maps2000/ag_map_with_alpha.txt.quartiles.tsv --alphas alpha_div.tsv --output alpha2000/
+./effect-size.py --mappings maps5000/ag_map_with_alpha.txt.quartiles.tsv --alphas alpha_div.tsv --output alpha5000/
```

Comment:
Can you add the commands to generate the beta diversity results?

Reply:
Not sure if you get an email. Anyway, just added them.